package.json

```diff
@@ -35,12 +35,11 @@
     "release": "auto shipit"
   },
   "dependencies": {
-    "@storybook/addons": "^7.0.0",
```

Comment:
@storybook/addons is no longer needed: https://www.npmjs.com/package/@storybook/addons